autonomy/cli/scaffold_fsm.py

```diff
@@ -192,8 +192,12 @@ def template_kwargs(self) -> Dict[str, str]:
             initial_states=_remove_quotes(str(self.dfa.start_states)),
             transition_function=_indent_wrapper(_remove_quotes(str(tf))),
             final_states=_remove_quotes(str(self.dfa.final_states)),
-            BaseBehaviourCls=self.abci_app_name.replace(ABCI_APP, BASE_BEHAVIOUR),
-            RoundBehaviourCls=self.abci_app_name.replace(ABCI_APP, ROUND_BEHAVIOUR),
+            BaseBehaviourCls=re.sub(
+                ABCI_APP, BASE_BEHAVIOUR, self.abci_app_name, flags=re.IGNORECASE
+            ),
+            RoundBehaviourCls=re.sub(
+                ABCI_APP, ROUND_BEHAVIOUR, self.abci_app_name, flags=re.IGNORECASE
+            ),
```

Comment:
re.sub and re.IGNORECASE because `reset_pause_abci` spells it as `ABCI` instead of `Abci`

https://github.com/valory-xyz/open-autonomy/blob/24c9d6aadb1a1b4e60f4a541f5212a1981cb6255/packages/valory/skills/reset_pause_abci/fsm_specification.yaml#L9

Reply:
Why not enforce this as a pattern for now? We already enforce quite a lot of convention here; seems arbitrary to deviate in this special case.

Reply:
I prefer that as well. I created a [Trello ticket](https://trello.com/c/r5LGI5J4/1302-fsmspecificationyaml-convention-checker-for-scaffold-fsm) for a small convention checker to be ran before scaffolding. This may become part of the cli tool using the DFA we already have perhaps.